invenio_checks/services/services.py

```diff
@@ -23,25 +37,101 @@ def rebuild_index(self, identity, uow=None):
         raise NotImplementedError()
 
 
+def get_check_config(id_):
+    """Get a check config by id."""
+    check_config = CheckConfig.query.get(id_)
+    if check_config == None:
+        raise ValueError(f"Check configuration with id '{id_}' not found")
+    return check_config
+
+
 class CheckConfigService(RecordService):
-    """Service for managing and check configurations."""
+    """Service for managing check configurations."""
+
+    @unit_of_work()
+    def create(self, identity, data, uow=None, **kwargs):
+        """Create a check configuration."""
+        self.require_permission(identity, "create")
+
+        schema = CheckConfigSchema()
+        try:
+            validated_data = schema.load(data)
+        except ValidationError as e:
+            raise ValueError(f"Validation error: {e.messages}")
```

Comment:
If there will be a form to create this, we will need to return a more structured error message, so that the UI can parse the response.